nav2_smac_planner/src/smac_planner_2d.cpp

```diff
@@ -340,45 +334,46 @@ nav_msgs::msg::Path SmacPlanner2D::createPlan(
   return plan;
 }
 
-void SmacPlanner2D::on_parameter_event_callback(
-  const rcl_interfaces::msg::ParameterEvent::SharedPtr event)
+rcl_interfaces::msg::SetParametersResult
+SmacPlanner2D::dynamicParametersCallback(std::vector<rclcpp::Parameter> parameters)
 {
+  auto result = rcl_interfaces::msg::SetParametersResult();
+  RCLCPP_INFO_STREAM(_logger, "Param callback, size: " << parameters.size());
```

Comment:
Please replace with `RCLCPP_INFO` call instead using `%i`. Its a stylistic thing, I like to keep logging statements consistent within a project such that we can easily scan through code and not have to think too much. This is really simple, but that's how the creep starts :-)

Reply:
Debug artefact, removing it